error while compiling

Hi compiling latest release on openSUSE Tumbleweed ends with error:

```
❯ ./configure && make                                                                                                                                           08:55:58
configure: loading site script /usr/share/site/x86_64-unknown-linux-gnu
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether make supports nested variables... (cached) yes
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether gcc understands -c and -o together... yes
checking for style of include used by make... GNU
checking dependency style of gcc... gcc3
checking how to run the C preprocessor... gcc -E
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for ANSI C header files... yes
checking for udev rules.d... disabled, classic SUID root mode
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating src/Makefile
config.status: creating config.h
config.status: config.h is unchanged
config.status: executing depfiles commands
make  all-recursive
make[1]: Entering directory '/home/dp/Downloads/light-1.2'
Making all in src
make[2]: Entering directory '/home/dp/Downloads/light-1.2/src'
  CCLD     light
/usr/lib64/gcc/x86_64-suse-linux/11/../../../../x86_64-suse-linux/bin/ld: light-light.o:/home/dp/Downloads/light-1.2/src/helpers.h:24: multiple definition of `light_loglevel'; light-main.o:/home/dp/Downloads/light-1.2/src/helpers.h:24: first defined here
/usr/lib64/gcc/x86_64-suse-linux/11/../../../../x86_64-suse-linux/bin/ld: light-helpers.o:/home/dp/Downloads/light-1.2/src/helpers.h:24: multiple definition of `light_loglevel'; light-main.o:/home/dp/Downloads/light-1.2/src/helpers.h:24: first defined here
/usr/lib64/gcc/x86_64-suse-linux/11/../../../../x86_64-suse-linux/bin/ld: impl/light-sysfs.o:/home/dp/Downloads/light-1.2/src/./helpers.h:24: multiple definition of `light_loglevel'; light-main.o:/home/dp/Downloads/light-1.2/src/helpers.h:24: first defined here
/usr/lib64/gcc/x86_64-suse-linux/11/../../../../x86_64-suse-linux/bin/ld: impl/light-util.o:/home/dp/Downloads/light-1.2/src/./helpers.h:24: multiple definition of `light_loglevel'; light-main.o:/home/dp/Downloads/light-1.2/src/helpers.h:24: first defined here
/usr/lib64/gcc/x86_64-suse-linux/11/../../../../x86_64-suse-linux/bin/ld: impl/light-razer.o:/home/dp/Downloads/light-1.2/src/./helpers.h:24: multiple definition of `light_loglevel'; light-main.o:/home/dp/Downloads/light-1.2/src/helpers.h:24: first defined here
collect2: error: ld returned 1 exit status
make[2]: *** [Makefile:352: light] Error 1
make[2]: Leaving directory '/home/dp/Downloads/light-1.2/src'
make[1]: *** [Makefile:475: all-recursive] Error 1
make[1]: Leaving directory '/home/dp/Downloads/light-1.2'
make: *** [Makefile:333: all] Error 2
```

any idea what is missing?

Thank you

--- a/src/helpers.c
+++ b/src/helpers.c
@@ -9,6 +9,7 @@
 #include <errno.h> // errno
 #include <libgen.h> // dirname 
 
+light_loglevel_t light_loglevel;
 
 bool light_file_read_uint64(char const *filename, uint64_t *val)
 {

--- a/src/helpers.h
+++ b/src/helpers.h
@@ -21,7 +21,7 @@ typedef enum {
     LIGHT_NOTE_LEVEL
 } light_loglevel_t;
 
-light_loglevel_t light_loglevel;
+extern light_loglevel_t light_loglevel;
 
 #define LIGHT_LOG(lvl, fp, fmt, args...)\
     if(light_loglevel >= lvl)\

